src/features/ListAllMovies/ui/MovieListBlock.tsx

```diff
@@ -15,8 +15,8 @@ export const MovieList: FC<MovieListProps> = () => {
     return (
         <div className="max-w-[1400px] block m-auto">
             <Tabs value={tabValue} className="ml-10">
-                <Tab onClick={() => setTabValue(0)} className="text-lg" label={t('whatsOn')}></Tab>
-                <Tab onClick={() => setTabValue(1)} label={t('whatsComing')}></Tab>
+                <Tab onClick={() => setTabValue(0)} style={{color: tabValue === 0 ? '#6e3996' : '', fontWeight: tabValue === 0 ? 'bold' : ''}} label={t('whatsOn')}></Tab>
```

Comment:
```suggestion
                <Tab onClick={() => setTabValue(0)} className={`${tabValue === 0 ? 'text-[#6e3996]' : ''} ${tabValue === 0 ? 'font-bold' : ''}`} label={t('whatsOn')}></Tab>
```

Or you can even install - https://www.npmjs.com/package/classnames And do like this className={classNames({ 'text-[#6e3996]': tabValue === 0, 'font-bold': tabValue === 0 })}